### PR TITLE
Fix: fixing some proptypes definitions

### PR DESCRIPTION
**Pull request by xiaofan2406**
I went through the media signoff process, and found these three places in adslot-ui are producing warnings.

As part of ADS-458

### CODE REVIEW

File: src/components/adslot-ui/AlertInput/index.jsx [hunk 1]
```diff
@@ -72,16 +72,14 @@ export default class AlertInput extends Component {
...
     } = this.props;
 
-    const className = classnames({
-      [baseClass]: true,
-      [alertStatus]: true,
-      'is-focused': this.state.isFocused,
-    });
+    const className = classnames(baseClass, { [alertStatus]: alertStatus, 'is-focused': this.state.isFocused });
+
+    const popoverClassName = classnames(`${baseClass}-popover`, { alertStatus });
```
Comment (mdotwills) [inline src/components/adslot-ui/AlertInput/index.jsx]: _we could probably get rid of this `baseClass` thing too to make the class strings searchable_

File: src/components/adslot-ui/AlertInput/index.jsx [hunk 2]
```diff
@@ -72,16 +72,14 @@ export default class AlertInput extends Component {
       placeholder,
       prefixAddon,
       suffixAddon,
-      alertStatus = '',
+      alertStatus,
```
Comment (sonhanguyen) [inline src/components/adslot-ui/AlertInput/index.jsx]: does this break anything? for example `alertStatus` being used as key down bellow?

Comment (xiaofan2406) [reply src/components/adslot-ui/AlertInput/index.jsx]: nay. It will be undefined. and `classnames` will know how to deal with falsy values.

Comment (xiaofan2406) [reply src/components/adslot-ui/AlertInput/index.jsx]: I manully tested on the adslot-ui doc page. It looks the same as before

**Comment by xiaofan2406** [comment]
yeah most of them. and some from the requestproductmodal as well

**Pull request merged**